### ISSUE TITLE
Building mfg-data fails

### ISSUE THREAD

**Issue by sethfischer** [sethfischer/atopile-action#1]
Building mfg-data fails

See [workflow run sethfischer/atopile-action@a1c8ffc](https://github.com/sethfischer/atopile-action/actions/runs/8059744213/job/22014659916).

Building mfg-data fails with:

```
  File "/opt/atopile/lib/python3.11/site-packages/atopile/manufacturing_data.py", line 84, in generate_manufacturing_data
    repo = git.Repo(config.get_project_context().project_path)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/atopile/lib/python3.11/site-packages/git/repo/base.py", line 276, in __init__
    raise InvalidGitRepositoryError(epath)
git.exc.InvalidGitRepositoryError: /github/workspace/tests/project-1
```

<details>
  <summary>Full error message</summary>

```
[04:44:16] INFO     Installing None in                             install.py:44
                    /github/workspace/tests/project-1                           
[04:44:19] INFO     Installing dependency generics                install.py:164
[04:44:20] INFO     Using ✨ v2.2.8 ✨ of generics                install.py:229
           INFO     Done! 🤙                                       install.py:69
[04:44:21] INFO     Using project /github/workspace/tests/project-1 common.py:73
           INFO     Building default                                 build.py:36
           INFO     Writing outputs to                               build.py:64
                    /github/workspace/tests/project-1/build                     
           INFO     Building copy-footprints                         build.py:77
           INFO     Building netlist                                 build.py:77
           INFO     Building bom                                     build.py:77
┏━━━━━━━━━┳━━━━━━━━━━━━┳━━━━━━━━━━━┳━━━━━━┓
┃ Comment ┃ Designator ┃ Footprint ┃ LCSC ┃
┡━━━━━━━━━╇━━━━━━━━━━━━╇━━━━━━━━━━━╇━━━━━━┩
└─────────┴────────────┴───────────┴──────┘
           INFO     Building designator-map                          build.py:77
┏━━━━━━━━━━━━━━┳━━━━━━┓
┃ Designator ↓ ┃ Name ┃
┡━━━━━━━━━━━━━━╇━━━━━━┩
└──────────────┴──────┘
┏━━━━━━━━┳━━━━━━━━━━━━┓
┃ Name ↓ ┃ Designator ┃
┡━━━━━━━━╇━━━━━━━━━━━━┩
└────────┴────────────┘
           INFO     Building mfg-data                                build.py:77
Traceback (most recent call last):
  File "/usr/local/bin/ato", line 8, in <module>
    sys.exit(cli())
             ^^^^^
  File "/opt/atopile/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/atopile/lib/python3.11/site-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/opt/atopile/lib/python3.11/site-packages/click/core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/atopile/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/atopile/lib/python3.11/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/atopile/lib/python3.11/site-packages/atopile/errors.py", line 270, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/opt/atopile/lib/python3.11/site-packages/atopile/cli/common.py", line 142, in wrapper
    return f(*args, **kwargs, build_ctxs=build_ctxs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/atopile/lib/python3.11/site-packages/atopile/cli/build.py", line 38, in build
    _do_build(build_ctx)
  File "/opt/atopile/lib/python3.11/site-packages/atopile/cli/build.py", line 79, in _do_build
    muster.targets[target_name](build_ctx)
  File "/opt/atopile/lib/python3.11/site-packages/atopile/cli/build.py", line 108, in wrapper
    return func(build_args)
           ^^^^^^^^^^^^^^^^
  File "/opt/atopile/lib/python3.11/site-packages/atopile/cli/build.py", line 155, in generate_manufacturing_data
    atopile.manufacturing_data.generate_manufacturing_data(build_ctx)
  File "/opt/atopile/lib/python3.11/site-packages/atopile/manufacturing_data.py", line 84, in generate_manufacturing_data
    repo = git.Repo(config.get_project_context().project_path)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/atopile/lib/python3.11/site-packages/git/repo/base.py", line 276, in __init__
    raise InvalidGitRepositoryError(epath)
git.exc.InvalidGitRepositoryError: /github/workspace/tests/project-1
```
</details>

**Comment by sethfischer** [comment]
Results of debugging by adding the following to `entrypoint.sh`:

```
ls -la
git status
```

Output below shows that `ls` can see the `.git/` directory and `git status` returns OK.
```
drwxr-xr-x 5 1001  127 4096 Feb 27 08:21 .
drwxr-xr-x 6 root root 4096 Feb 27 08:22 ..
drwxr-xr-x 8 1001  127 4096 Feb 27 08:21 .git
drwxr-xr-x 3 1001  127 4096 Feb 27 08:21 .github
-rw-r--r-- 1 1001  127  260 Feb 27 08:21 Dockerfile
-rw-r--r-- 1 1001  127 1056 Feb 27 08:21 LICENSE
-rw-r--r-- 1 1001  127  384 Feb 27 08:21 README.md
-rw-r--r-- 1 1001  127  280 Feb 27 08:21 action.yml
-rwxr-xr-x 1 1001  127  210 Feb 27 08:21 entrypoint.sh
drwxr-xr-x 3 1001  127 4096 Feb 27 08:21 tests
On branch fix-build-mfg-data
Your branch is up to date with 'origin/fix-build-mfg-data'.
```

**Comment by sethfischer** [comment]
See atopile/atopile#184

**Comment by sethfischer** [comment]
Fixed with atopile/atopile#184.
